libs/game/scene.ts

```diff
@@ -115,12 +115,5 @@ namespace scene {
             // update screen
             this.eventContext.registerFrameHandler(200, control.__screen.update);
         }
-
-        addSprite(sprite: SpriteLike) {
```

Comment:
hummm. i think we can to keep this. spriteCount and "allSprites" are internal field to this class.